demonstrations/tutorial_vqe_uccsd_obs.py

```diff
@@ -0,0 +1,253 @@
+r"""
+VQE simulations in different sectors of the spin quantum number
+===============================================================
+
+.. meta::
+    :property="og:description": Find the lowest-energy states of a Hamiltonian in different
+        sector of the spin quantum number using the variational quantum eigensolver
+        algorithm in PennyLane.
+    :property="og:image": https://pennylane.ai/qml/_images/pes_h2.png
+
+Quantum computers offer a promising avenue to perform first-principles simulations of the
+electronic structure of molecules and materials that are currently intractable using classical
+high-performance computers. In particular, the Variational Quantum Eigensolver (VQE) algorithm
+:ref:`[1, 2]<vqe_references>` has proven to be a valuable quantum-classical computational
+approach to find the lowest-energy eigenstate of the electronic Hamiltonian by using Noisy
+Intermediate-Scale Quantum (NISQ) devices. (WILL ADD SOME REFERENCES HERE)
+
+In the absence of `spin-orbit coupling <https://en.wikipedia.org/wiki/Spin-orbit_interaction>`_ the
+electronic Hamiltonian matrix is block diagonal in the total spin quantum numbers. In other words,
+one can expand the many-electron wave function of the molecule as a linear
+combination of `Slater determinants <https://en.wikipedia.org/wiki/Slater_determinant>`_
+with the same total-spin projection :math:`S_z`, and diagonalize the Hamiltonian in this basis
+to obtain the energy spectrum in this particular subspace. For example, the figure below
+shows the energy spectra of the Hydrogen molecule calculated in different spin sectors. Notice,
+that the ground state with energy :math:`E_\mathrm{gs}=-1.136189` Ha has spin quantum numbers
+:math:`S=0`, :math:`S_z=0` while the lowest-lying excited states, with energy
+:math:`E^*=-0.478453` Ha, show a three-fold spin degeneracy with quantum numbers
+:math:`S=1` and :math:`S_z=0, \pm 1`.
```

Comment:
```suggestion
:math:`S=1` and :math:`S_z=`1, 0, - 1`.
```
makes it clearer in connection with the figure